### PR TITLE
fix(budget-detail-page): fix style of header

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
image

### CODE REVIEW

File: apps/web/src/services/cost-explorer/budget/budget-detail/BudgetDetailPage.vue [hunk 1]
```diff
@@ -128,21 +129,29 @@ const handleConfirmDelete = () => {
...
-        @apply inline-flex items-center;
-        line-height: 1.09375rem;
-        float: right;
-        .divider {
-            height: 0.875rem;
-            margin: 0 0.5rem;
+    .header-container {
+        /* custom design-system component - p-header */
+        &:deep(.heading-wrapper) {
+            @apply flex items-center;
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/budget/budget-detail/BudgetDetailPage.vue:135]: issue: design system style custom is the last thing we have to do.
image
image

You can fix it without using deep by adding min-height to .title-right-extra.

**Review by WANZARGEN — APPROVED** [review]
I suggested the better way without customizing design system style!

File: apps/web/src/services/cost-explorer/budget/budget-detail/BudgetDetailPage.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/services/cost-explorer/budget/budget-detail/BudgetDetailPage.vue:135]: Holy.. Thank you genius.

**Review by skdud4659 — APPROVED** [review]
LGTM~

**Pull request merged**